GitStartHQ/tech-screen-calcom
# feat: added drag and drop in profile picture update

## What does this PR do?

This PR adds a drag and drop feature to the profile picture update. Users can now drag and drop an image onto the designated area to upload a profile picture. The existing "choose files" feature remains available for users who prefer to select an image using the traditional file picker. The provides users with an alternative and more intuitive way of uploading their profile pictures. 

Fixes [#calcom9199](https://github.com/calcom/cal.com/issues/9199)

 Loom Video: https://www.loom.com/share/59e27a13c41c4c5cb7a766c15195e3e3?sid=77e50682-2cbd-465c-853a-9c293f9d7edb

## Type of change

- New feature (non-breaking change which adds functionality)

## How should this be tested?

It can be tested using either the _**"add profile picture"**_ button in the getting started section or the _**"change avatar"**_ button in the settings section. When either of the buttons are clicked, a modal appears and you can drag and drop your desired photo into the circular gray area with the dashed borders. After saving, your avatar should appear as normal.


## Checklist
